testsuite/indexer_grpc_local.py

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/env python3
```

Comment:
Is this file using strict typing?

Reply:
nope. I'm planning on doing that for all scripts in `testsuite` soon...